src/lxc/attach.c

```diff
@@ -66,6 +66,7 @@
 #include "lsm/lsm.h"
 #include "lxclock.h"
 #include "lxcseccomp.h"
+#include "macro.h"
```

Comment:
Should probably be part of the `macro: move macros from attach.c` commit as I see no relation to `rexit()`.